readme.md

```diff
@@ -7,16 +7,16 @@ The RPTPN is a specification to allow for chatting, file sharing, code sharing,
 ## Spec
 How stuff works
 
-This was inspired by the dead Replit Blog Chat protocol me and Codemonkey51 started thinking up before the entire idea of a chat within the Replit Blog's logs died down. This is my best attempt at revitalizing it.
+This was inspired by the dead Replit Blog Chat protocol Codemonkey51 and I started thinking up before the entire idea of a chat within the Replit Blog's logs died down. This is my best attempt at reviving it.
 
 ### Repl Name
-To keep everything simplified, every repl running the protocol should be named rptpn. This simplifies storing information on other peers, which just makes evertything easier and much more straight forward.
+To keep everything simplified, every repl running the protocol should be named `rptpn`. This simplifies storing information on other peers, which just makes everything easier and much more straight forward.
```

Comment:
"RPTPN" isn't code, I don't think it should be written as such; I'd also suggest that "RPtPN" be used in terms of capitalisation.

Reply:
Whoops I didn't read that correctly, that's fine, maybe note that it can be case-insensitive.

Reply:
Oops - pushed a commit to fix that

Reply:
Yay